src/main/resources/templates/table.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
```

Comment:
[I see you've applied TailwindCSS.](https://github.com/herotechcourse/spring-ecommerce-product/pull/6/commits/6f0c99d1ccd0bd7eee9c70e91906f44abfbde77c) 😮 
I'm guessing you or the pair had some experience with it?